src/containers/LoginPage.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react'
+import NavBar from "../components/NavBar";
+import SignInForm from "../components/LogInForm";
+
+
+export class LoginPage extends Component {
+  render() {
+    return (
+      <div>
+          <NavBar/>
```

Comment:
Why not move it into `App` level? Why do you copy paste it in every subcomponents?